javascript/express/sql_injection/.snapshots/sql_injection_juice.yml

```diff
@@ -1,61 +1,2 @@
-high:
```

Comment:
I need to come back to this one since it should still fail.

Reply:
Related: https://github.com/Bearer/bearer/issues/973